packages/mwp-app-render/src/components/Dom.jsx

```diff
@@ -2,6 +2,8 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import escapeHtml from 'escape-html';
 
+import { polyfillServiceUrl } from 'mwp-app-render/lib/components/PageWrap/browserPolyfill';
```

Comment:
If we go with this approach, I think it makes sense to move the browserPolyfill out of PageWrap and next to Dom.jsx